docs/cms/developers/publishing/index.md

```diff
@@ -2,17 +2,17 @@
 
 ## Overview
 
-When content (like pages or assets) is published from actions available in the authoring UI, it is getting copied from `/content` to `/published` content tree.
+When content (such as pages or assets) is published via actions available in the authoring UI, it is copied from `/content` to `/published` within the content tree.
 
-Content references (paths save in JCR) are updated (in same way as when pages are copied/moved) automatically during publishing.
-So published content reference other published content items after publishing.
+Content references (paths saved in JCR) are automatically updated (just as they would be when pages are copied or moved) during publishing.
+This means that published content references other published content items after publishing.
 
-Publishing API allows to get information about status of content and publish/unpublish the content.
-SPI interfaces allows to take actions before/after publishing/unpublishing and customize published content if needed.
+The publishing API allows you to get information about the status of content and to publish/unpublish content.
+SPI interfaces allow you to take actions before publishing, after publishing and during unpublishing. You can also customize published content if needed.
```

Comment:
```suggestion
SPI interfaces allow you to take actions before publishing, after publishing, before unpublishing, and after unpublishing. You can also customize published content if needed.
```

Reply:
The previous version sounded a bit unclear, to me. I understood it as if the action was being performed precisely at the time of unpublishing.